src/main/java/core/basesyntax/Printable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Printable {
+    void printFigureInfo();
+}
```

Comment:
Let it be `Drawable` and `draw` correspondingly as mentioned in the requirements. (... able to draw it. To 'draw' means ...).